From 38d276c7a60086bcb58f62262daf461660f1c6d5 Mon Sep 17 00:00:00 2001
From: Jason Montleon <jmontleo@redhat.com>
Date: Tue, 26 Nov 2024 20:56:43 -0500
Subject: [PATCH 1/4] update pwm_apply_state to pwm_apply_might_sleep

---
 drivers/hwmon/eswin-fan-control.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/hwmon/eswin-fan-control.c b/drivers/hwmon/eswin-fan-control.c
index aff50deb2740..01447e641fa3 100644
--- a/drivers/hwmon/eswin-fan-control.c
+++ b/drivers/hwmon/eswin-fan-control.c
@@ -139,7 +139,7 @@ static ssize_t eswin_fan_pwm_ctl_store(struct device *dev, struct device_attribu
 		dev_err(dev, "get error attr index 0x%x\n", attr->index);
 	}
 
-	pwm_apply_state(ctl->pwm, &state);
+	pwm_apply_might_sleep(ctl->pwm, &state);
 
 	return count;
 }
@@ -240,7 +240,7 @@ static int eswin_fan_control_set_pwm_duty(const long val, struct eswin_fan_contr
 
 	pwm_get_state(ctl->pwm, &state);
 	pwm_set_relative_duty_cycle(&state, val, 100);
-	pwm_apply_state(ctl->pwm, &state);
+	pwm_apply_might_sleep(ctl->pwm, &state);
 
 	return 0;
 }
@@ -513,7 +513,7 @@ static int eswin_fan_control_probe(struct platform_device *pdev)
 	state.duty_cycle = state.period/2;
 	dev_err(&pdev->dev, "state.period: %d state.duty_cycle: %d\n",
 			state.period,state.duty_cycle);
-	ret = pwm_apply_state(ctl->pwm, &state);
+	ret = pwm_apply_might_sleep(ctl->pwm, &state);
 	if (ret) {
 		dev_err(&pdev->dev, "failed to apply initial PWM state: %d\n",
 			ret);
-- 
2.47.0

