From eae9ce99d83e378c2ca1c1821c5178cf67b420bc Mon Sep 17 00:00:00 2001
From: Jason Montleon <jmontleo@redhat.com>
Date: Wed, 27 Nov 2024 02:54:49 -0500
Subject: [PATCH 218/222] rename cpr_val to cpr_value

---
 drivers/tty/serial/8250/8250_dw.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/tty/serial/8250/8250_dw.c b/drivers/tty/serial/8250/8250_dw.c
index 629466a07c6a..e82300aa21cc 100644
--- a/drivers/tty/serial/8250/8250_dw.c
+++ b/drivers/tty/serial/8250/8250_dw.c
@@ -767,7 +767,7 @@ static const struct dev_pm_ops dw8250_pm_ops = {
 
 static const struct dw8250_platform_data dw8250_dw_apb = {
 	.usr_reg = DW_UART_USR,
-	.cpr_val = 0x22232,
+	.cpr_value = 0x22232,
 };
 
 static const struct dw8250_platform_data dw8250_octeon_3860_data = {
-- 
2.47.0

